pkg/run/v0/ibmcloud/ibmcloud.go

```diff
@@ -76,11 +114,29 @@ func Run(scope scope.Scope, name plugin.Name,
 		authInstPlugin = ibmcloud_auth_inst.NewVolumeAuthPlugin(options.Username, options.APIKey, options.VolumeAuth.VolumeID)
 	}
 
+	l4Map := map[string]loadbalancer.L4{}
+	for _, lb := range options.LBs {
+		var lbPlugin loadbalancer.L4
+		lbPlugin, err = ibmcloud_loadbalancer.NewIBMCloudLBPlugin(options.Username, options.APIKey, lb.LBName, lb.LBUUID)
+		if err != nil {
+			return
```

Comment:
Should we log an error here?

Reply:
I find the logging inconsistent across plugins, so adding it here.